Automattic/jetpack
# Connection: Add additional valid "source" value to the Jetpack connection URL for A4A

Related to https://github.com/Automattic/jetpack-genesis/issues/290

## Proposed changes:

* Adds `a8c-for-agencies` as a valid `source` query parameter value, alongside `jetpack-manage`.
* Updates tests.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack-genesis/issues/290

https://github.com/Automattic/jetpack/pull/33984

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Go to Jetpack Live Branch with Jetpack
* Append the URL with wp-admin/admin.php?page=jetpack&connect_url_redirect=true&jetpack_connect_login_redirect=true&source=a8c-for-agencies
* Verify the link you are redirected(/jetpack/connect/authorize) to has source=a8c-for-agencies in it.
